src/sum-of-odd.js

```diff
@@ -25,7 +25,17 @@
  * @return {number}
  */
 function rowSumOddNumbers(row) {
-  // write code here
+  let startNumberOfRow = row * row - (row - 1);
+  let sum = startNumberOfRow;
+  if (row < 0) {
+    return 0;
+  } else {
```

Comment:
no need for this else
```suggestion
  }
```